src/main/haskell/kore/src/Kore/Step/Simplification/Bottom.hs

```diff
@@ -0,0 +1,31 @@
+{-|
+Module      : Kore.Simplification.Bottom
+Description : Tools for Bottom pattern simplification.
+Copyright   : (c) Runtime Verification, 2018
+License     : UIUC/NCSA
+Maintainer  : virgil.serbanuta@runtimeverification.com
+Stability   : experimental
+Portability : portable
+-}
+module Kore.Step.Simplification.Bottom
+    ( simplify
+    ) where
+
+import           Kore.AST.Common
+                 ( Bottom (..) )
+import           Kore.Step.OrOfExpandedPattern
+                 ( OrOfExpandedPattern )
+import qualified Kore.Step.OrOfExpandedPattern as OrOfExpandedPattern
+                 ( make )
+import           Kore.Step.Simplification.Data
+                 ( SimplificationProof (..) )
+
+{-| simplifies a Bottom pattern, which means returning an always-false or.
```

Comment:
Because of the global instance coherence constraint, creating a class works best if there is only one valid instance you can make for a type, or at least one canonical instance. I don't think we can say there is "one valid simplifier" you can define for a type. I'm not even sure there is a clear "canonical simplifier" for each type. I can imagine that someone might want to substitute their own simplifier in the future, which would be ~~impossible~~ fairly difficult if it was a class. But feel free to tell me, "No, that's silly, nobody would ever want to define their own simplifiers!" :grinning: